Source/JavaScriptCore/Scripts/postprocess-header-rule

```diff
@@ -43,11 +38,17 @@ function process_definitions () {
     source "${DEFINITIONS_FILE}"
 }
 
-function rewrite_headers () {
-    local SOURCE_FILE="${1}"
-    local WORK_FILE="${2}.tmp"
-    local DEST_FILE="${2}"
+DEFINITIONS_PATH=usr/local/include/WebKitAdditions/Scripts/postprocess-framework-headers-definitions
+
+process_definitions "${BUILT_PRODUCTS_DIR}/${DEFINITIONS_PATH}" || process_definitions "${SDKROOT}/${DEFINITIONS_PATH}"
+
+# For historical compatibility, JSC's postprocessing may be toggled separately
```

Comment:
I'm not sure what this means. People certainly build JSC separately, but that's not historical, so I must be thinking in the wrong direction.